File: STC/git_testing/prog.c
#include <stdio.h>
#define N 1000000

void isqrt (float *a, int n);

int main(){
    float a[N];
    int i;
        for (i=0; i<N; i++) a[i]=1.0e0 + i;
        isqrt(a,N);
        printf("N a(0) a(N-1) : %d %f %f\n", N,a[0],a[N-1]);
}

File: STC/git_testing/prog.f90
program invsqrt
integer, parameter :: N=1000000
real a(N);
    do i=1,N; 
        a(i)=i; 
    end do

    call isqrt(a,N);
    print*,"N a(1) a(N): ", N,a(1),a(N)

end program
